src/assets/locales/fr.json

```diff
@@ -310,8 +310,8 @@
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "more-menu.you": "Vous",
-  "more-menu.you-signed-in-as": "Vous (connecté(e) en tant que {email})",
+  "more-menu.not-signed-in": "Vous n'êtes pas connecté",
```

Comment:
Ah I forgot, to be inclusive, "Vous n'êtes pas connecté(e)."

Reply:
Thanks so much for your help, updated.